src/view/pages/gettingStarted.tsx

```diff
@@ -87,38 +132,83 @@ export class GettingStartedPage extends React.Component {
                             </a>
                         </h3>
                     </div>
-                    <div id="CPX" className="inv">
-                        <h2> Tutorial for CPX </h2>
+                    <div id="CLUE" className="inv">
+                        <h2> Tutorial for CLUE </h2>
                         <h3>
-                            1. Import the micro:bit library to use it (This is
+                            1. Import the the main CLUE library (This is
                             required)
                         </h3>
                         <span className="codeBox">
-                            <pre>from adafruit_circuitplayground import cp</pre>
+                            <pre>from adafruit_clue import clue</pre>
                         </span>
-                        <h3> 2. Turn on the little red LED</h3>
+                        <h3>
+                            {" "}
+                            2. Display text on the CLUE and change the text when
+                            a button is pressed
+                        </h3>
                         <span className="codeBox">
+                            <pre>
+                                clue_data =
+                                clue.simple_text_display(title="CLUE!",
+                                text_scale=2)
+                            </pre>
                             <pre>while True:</pre>
-                            <pre> cp.red_led = True</pre>
+                            <pre> clue_data[1].text = "Hello World!"</pre>
+                            <pre>
+                                {" "}

```

Comment:
Is the space here necessary?

Reply:
I think prettier added these weird spaces. It also ruined the tabulation so I removed prettier for these sections :)

Reply:
Should be good now!